tests/test_util.py

```diff
@@ -45,3 +54,15 @@ def test_warn(capsys):
     util.warn('test warn')
     _, err = capsys.readouterr()
     assert '!!! WARNING: test warn' in err
+
+
+def test_import_app():
+    assert util.import_app("test:app")
+
+    with pytest.raises(ImportError) as e_info:
+        util.import_app("a:app")
+    assert "No module" in str(e_info)
+    
+    with pytest.raises(AppImportError) as e_info:
+        util.import_app("test:wrong_app")
+    assert "Failed to " in str(e_info)
```

Comment:
Can we check for `"Failed to find application object 'wrong_app' in 'test'"` instead of `"Failed to"`.

Reply:
Sure!